File: sdks/rust/tests/compile_fail/invalid.rs
use speechswitch_types::{generated::{hume, xai}, runtime::StreamingInput};

fn wrong_model(request: &mut hume::TtsRequestOctave2TextVoice) {
    request.instructions = Some("Whisper".to_string());
}
fn wrong_literal() {
    let _: xai::TtsRequestStreamingTextTextItemClearCommand = "cancel";
}
fn wrong_stream(value: StreamingInput<xai::TtsRequestStreamingTextTextItem>) -> StreamingInput<String> {
    value
}

File: sdks/go/testdata/invalid/invalid.go
package invalid

import (
    "github.com/speechswitch/client/sdks/go/generated/hume"
    "github.com/speechswitch/client/sdks/go/generated/xai"
    "github.com/speechswitch/client/sdks/go/runtime"
)

func wrongModel(request *hume.TtsRequestOctave2TextVoice) {
    request.Instructions = "Whisper"
}
var wrongLiteral xai.TtsRequestStreamingTextTextItemClearCommand = "cancel"
func wrongStream(input runtime.Input[xai.TtsRequestStreamingTextTextItem]) runtime.Input[string] {
    return input
}
